lib/govuk_content_schemas/schema_combiner.rb

```diff
@@ -60,6 +74,20 @@ def add_format_field(schema)
     end
   end
 
+  # v2 schemas do not require these properties.
+  def update_required_properties(schema)
+    return if schemas.key?(:metadata) && schemas.key?(:base_links) # v1 schema
+
+    if schema['required']
```

Comment:
I don't think this logic really belongs in the combiner script. Is there any way we can move it into the schemas themselves?